.github/workflows/azure-static-web-apps-gentle-beach-0eb2d061e.yml

```diff
@@ -0,0 +1,50 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - Assignment1
+    paths:
+      - "wordle-web/**"
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - Assignment1
+  workflow_dispatch:
+
+jobs:
+  build_and_deploy_job:
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - name: Get Code
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          lfs: false
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v4.0.2
+        with:
+          # Version Spec of the version to use. Examples: 12.x, 10.15.1, >=10.15.0.
+          node-version: 18
+      - name: Install Packages
+        run: npm ci
+        working-directory: ./wordle-web
+      - name: Generate
+        run: npm run generate
+        working-directory: ./wordle-web
+      #- name: Run Vitest

```

Comment:
Un-commenting this will still allow you to run tests when you push to your own branch, even though we don't have testing permissions for pull-requests.  This will allow you to make sure your tests pass when deployed on azure once pushed.

Reply:
Good call, can't even remember why it was commented out in the first place.